oidc-ui/src/App.css

```
@@ -280,3 +280,52 @@ input[type="password"]::-ms-clear {
+  }
+}
+
+/* consent timer */
+.consent-timer {
+  margin: -16px -32px 0px !important;
+  background: #fff9f0 !important;
+  padding: 16px !important;
+  border-top-left-radius: var(--thunderid-border-radius-large) !important;
+  border-top-right-radius: var(--thunderid-border-radius-large) !important;
+
+  font-weight: 500 !important;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)